docs/mkdocs/mkdocs.yml

```diff
@@ -226,3 +226,4 @@ nav:
       - UEFI: 'pkg/uefi/rpi/README.md'
       - WLAN: 'pkg/wlan/FIXME.md'
       - 'Clock Synchronization': 'docs/CLOCK-SYNCHRONIZATION.md'
+      - 'EVE Memory Settings': 'docs/MEMORY-SETTINGS.md'
```

Comment:
Would it make sense to create section `Memory` (like we have `Networking`, `Security`, etc.) and put there link to this doc + `memory-monitor/README.md` + later `psi-*/README.md`?

Reply:
Yep. Will do.

Reply:
@rene will have to remind us both :)